guides/v2.0/install-gde/prereq/security.md

```diff
@@ -20,12 +20,18 @@ redirect_from: /guides/v1.0/install-gde/prereq/security.html
 ### Suggestion for installing the Magento software with Apache
 If you choose to enable SELinux, you might have issues running the installer unless you change the *security context* of some directories as follows:
 
-	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/app/etc
-	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/var
-	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/media
-	chcon -R --type httpd_sys_rw_content_t <your Magento install dir>/pub/static
+	export mageroot="/path/to/your_magento_install_dir"
```

Comment:
All these commands should be wrapped in code fences and separated by `&&` so you can copy/paste the whole block and run it in a terminal.
```
>```bash
>export mageroot="/path/to/your_magento_install_dir"
>...
>```
```